Tribler/community/market/wallet/btc_wallet.py

```diff
@@ -0,0 +1,257 @@
+import os
+import sys
+from threading import Thread
+
+import keyring
+from twisted.internet.defer import Deferred, succeed, fail, inlineCallbacks
+from twisted.internet.task import LoopingCall
+
+import Tribler
+
+# Make sure we can find the electrum wallet
+from Tribler.community.market.wallet.wallet import InsufficientFunds, Wallet
+
+sys.path.append(os.path.join(os.path.dirname(os.path.abspath(Tribler.__file__)), '..', 'electrum'))
+
+import imp
+imp.load_module('electrum', *imp.find_module('lib'))
```

Comment:
Unfortunately, I can't think of a better solution. In the future, I would recommend to use the Armory wallet which seems to have a Python interface (https://www.bitcoinarmory.com/using-armory-python/, suggested by @vandenheuvel).